util/logging.go

```diff
@@ -165,6 +186,9 @@ func logSanitizedHeader(input_header http.Header) {
 
 // Logs the essentials of a HTTP request
 func ProcessRequestOutput(caller, operation, url, payload string, req *http.Request) {
+	if EnableHttpStack {
+		GovcdHttpStack.Push(OperationRec{OpType: "request", Caller: caller, Req: req, Data: payload, Url: url, Operation: operation})
```

Comment:
Does this use log santitized header? If it doesn't, doesn't it have the potential to log confidential passwords

Reply:
The header is sanitized when the HTTP operation is dumped, as it uses the same function as the regular logging.

Reply:
Doesn't it call this function later in the method though. Wouldn't those calls be redundant then.

Reply:
When we dump the HTTP records, `EnableHttpStack` gets disabled, to avoid an endless loop.

Reply:
ahh ok that makes more sense